manifests/params.pp

```diff
@@ -17,21 +17,29 @@
   $version = 'latest'
   $service = 'jenkins'
   $install_from_git = false
+  $install_from_pip = true
+  $install_from_pkg = false
   $git_revision     = 'master'
   $git_url          = 'https://git.openstack.org/openstack-infra/jenkins-job-builder'
 
   case $::osfamily {
     'RedHat', 'Amazon': {
-      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse']
+      if $::operatingsystemrelease =~ /^6/ {
+        $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse', 'PyYAML']
+        # This requires the dcaro/jenkins-job-builder repository
+        $jjb_packages    = [ 'jenkins-job-builder']
+      } else {
+        $python_packages = [ 'python', 'python-devel', 'python-pip', 'PyYAML']
```

Comment:
Turns out it was as simple as the case differing between the RPM and the pip packages ... our spec tests are passing now, will test on our dev boxes as well.